config/logstash.yml

```diff
@@ -85,6 +85,11 @@
 #
 # config.debug: false
 #
+# When enabled, process escaped characters such as \n and \" in strings in the
+# pipeline configuration files.
+#
+# config.strings.support_escapes: false
```

Comment:
Why not just `config.support_escapes: false` This applies to both `config.strings` i.e. `-e` as well as `-f` right?